tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -75,7 +75,7 @@ def test_serialize_model(self):
 
             fixture_name = 'sklearn.tree.tree.DecisionTreeClassifier'
             fixture_short_name = 'sklearn.DecisionTreeClassifier'
-            fixture_description = 'Automatically created scikit-learn flow.'
+            fixture_description = self.extension._get_sklearn_description(model)
```

Comment:
Please do not test the function(s) by calling the function it relies on itself. Is it possible that you put the actual content of the docstring here?

Reply:
Agreed. However, `_get_sklearn_description` is not a function being directly unit tested. 
I can always paste the 1024 character length string. 
I'll do that if you say so.

Reply:
Not directly, but it does the work under the hood, right? If yes, I'd prefer if you paste the string here.